File: src/main/java/no/nav/tjenestepensjon/simulering/service/SimulerPensjonService.java
package no.nav.tjenestepensjon.simulering.service;

import no.nav.tjenestepensjon.simulering.domain.TPOrdning;

import java.util.List;

public interface SimulerPensjonService {

    SimulerPensjonResponse simulerPensjon(List<TPOrdning> tpOrdningList, TPOrdning tpLatest);
}

File: src/main/kotlin/no/nav/tjenestepensjon/simulering/service/SimulerPensjonService.kt
package no.nav.tjenestepensjon.simulering.service

import no.nav.tjenestepensjon.simulering.model.domain.TPOrdning

interface SimulerPensjonService {
    fun simulerPensjon(tpOrdningList: List<TPOrdning>, tpLatest: TPOrdning): SimulerPensjonResponse
}